apps/extension/src/application/trading-copilot/subscriptions-management/components/subscriptions-list/subscription-item.tsx

```diff
@@ -63,44 +69,62 @@ export const SubscriptionItem = ({ subscription, onRemove }: Properties) => {
       <div className="flex items-center">
         <LazyImage
           src={avatarQuery.data}
-          className="size-5 rounded-full"
+          className="size-8 rounded-full border border-[#DBDDE2] bg-neutral-200"
           fallbackComponent={
-            <Icon size={20} name="PersonIcon" className="rounded-full" />
+            <div className="flex size-8 items-center justify-center rounded-full border border-[#DBDDE2] bg-neutral-200">
+              <IdrissIcon
+                size={20}
+                name="CircleUserRound"
+                className="m-auto text-neutral-700"
```

Comment:
we should use `text-neutral-500` to match the designs
![image](https://github.com/user-attachments/assets/43119680-c3b7-4d89-a352-47b3089d3c7d)